package com.kantek.coroutines.exceptions

class ResourceException(val resource: Int) : Throwable()
